docs/src/modules/components/bootstrap.js

```diff
@@ -1,7 +1,3 @@
-// Use the same helper as Babel to avoid bundle bloat.
-import 'core-js/modules/es6.array.find-index';
-import 'core-js/modules/es6.set';
```

Comment:
This created a different environment depending on the bundle. You can verify this by adding a `Symbol` polyfill. React elements will have sometimes types that are numbers, sometimes Symbols depending on the time and place they were created.